SampleStyleGuide.md

```diff
@@ -0,0 +1,114 @@
+# Sample Style Guide
+## Technical Writing Academy - May 2021 <br>
+--------
+
+## Table of contents<br>
+
+[Bold](#Bold)
+
+[Italics](#Italics)
+
+[Fixed Font](#FixedFont)
+
+[Abbreviations](#Abbreviations)
+
+[Capitalization](#Capitalization)
+
+[Acronyms](#Acronyms)
+
+[Commas in Lists](#CommasInLists)
+
+[Numbers](#Numbers)
+
+[Contraction](#Contraction)
+
+[Apostrophes](#Apostrophes)
```

Comment:
Could you create either a table or a list for this?

Reply:
Done! And I also changed the repository description. :)